pkg/doc/verifiable/credential.go

```diff
@@ -319,6 +367,32 @@ func WithPublicKeyFetcher(fetcher PublicKeyFetcher) CredentialOpt {
 	}
 }
 
+// WithNoCustomJSONSchemaCache option is used for enabling custom JSON schema caching.
+func WithNoCustomJSONSchemaCache() CredentialOpt {
+	return func(opts *credentialOpts) {
+		opts.noCustomSchemaCache = true
+	}
+}
+
+// ClearCustomJSONSchemasCache clears custom JSON schema cache.
+func ClearCustomJSONSchemasCache() {
+	jsonSchemaCache.clear()
+}
+
+// JSONSchemaCacheParam defines JSON schema cache parameters like expiration and cache size limit.
+type JSONSchemaCacheParam struct {
+	Expiration time.Duration
+	Size       int
+}
+
+// ConfigureCustomJSONSchemasCache defines JSON schema cache parameters.
+func ConfigureCustomJSONSchemasCache(param JSONSchemaCacheParam) {
```

Comment:
this function looks dangerous. Mutations without locks.

it would be best to refactor to remove globals. at that point, an instance could be constructed and I doubt you would need a configure function.

Reply:
The schema cache is expected to be used by multiple invocations of `verifiable.NewCredential()`. There is no sense to keep the cache inside `verifiable.Credential` struct as we use downloaded schema content only once while decoding VC. Please advise @troyronda  how to avoid globals in this case.

Reply:
Globals are removed. Thanks @troyronda !

Reply:
@kdimak I still see a "noglobals" in one of the files. Not sure if that one should still be there - just wanted to give you a heads up to double check :)